aten/src/ATen/native/mkldnn/MKLDNNConversions.cpp

```diff
@@ -280,6 +281,48 @@ static Tensor mkldnn_reorder_linear_weight(
   return new_with_itensor_mkldnn(std::move(result), optTypeMetaToScalarType(self.options().dtype_opt()), self.options().device_opt());
 }
 
+Tensor pack_linear(const Tensor& weight, const Tensor& input) {
+#if !defined(__aarch64__) || !AT_MKLDNN_ACL_ENABLED()
```

Comment:
Can you please explain, why it needs to be limited to aarch64 and ACL instead of enabled whenever mkldnn is used?